lib/screens/addLocation.js

```diff
@@ -0,0 +1,78 @@
+
+import React, { Component } from 'react';
+import { StyleSheet, View, ScrollView, Button } from 'react-native';
+import Dropdown from './dropdown';
+
+export default class AddLocation extends Component {
+  constructor() {
+    super();
+    this.state = {
+      ViewArray: []
+    }
+    this.Array_Value_Index = 2;
+  }
+  Add_New_View_Function = () => {
+    let New_Added_View_Value = { Array_Value_Index: this.Array_Value_Index }
+    this.setState({ ViewArray: [...this.state.ViewArray, New_Added_View_Value] }, () => {
+      this.Array_Value_Index++;
+    });
+  }
+  render() {
+    let Render_Animated_View = this.state.ViewArray.map((item, key) => {
+      return (
+        <View
+          key={key}
+          style={{ paddingTop: 5, height: 50 }}
+        >
+          <Dropdown placeholder=' End Location' />
+        </View>
+      );
+    });
+
+    return (
+      <View>
+        <ScrollView style={styles.limitScrollContainer}>
+          <View style={styles.dropdownContainer}>
+            <Dropdown placeholder=' Start Location'
+            //defaultValue= get user's current location
```

Comment:
```suggestion
```
Don't think we need this

Reply:
I was thinking this would be something useful, like to get a default value for the starting location text box to be the users current location. Similar to how we currently have set default depart time to current time